docs/guides/deleting-series.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Deleting Series"
+linkTitle: "Deleting Series"
+weight: 3
+slug: deleting-series
+---
+
+:warning: This feature is currently experimental and is only supported for Chunks storage. :warning:
+
+Cortex supports deletion of series using [Prometheus compatible API](https://prometheus.io/docs/prometheus/latest/querying/api/#delete-series).
+It however does not support [Prometheuses Clean Tombstones](https://prometheus.io/docs/prometheus/latest/querying/api/#clean-tombstones) API by trading it in for another feature for allowing cancellation of delete requests.
```

Comment:
Is "Prometheuses" intentional? Or should be "Prometheus'"

Reply:
Not sure "by trading it in for another feature for allowing cancellation of delete requests" is easy to understand. Maybe we could just say "because Cortex uses a different mechanism to manage deletions".

Reply:
>Is "Prometheuses" intentional? Or should be "Prometheus'"
Yes, it is intentional. I can remove it if it looks weird.